spec/models/review_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+describe Review do
+  describe "assosiasions" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.